src/components/chat/chatMixin.js

```diff
@@ -76,6 +78,49 @@ export default {
                     this.historyLoading = false;
                 });
         },
+        /*
```

Comment:
might want to delete this block of comments if it's not relevant, or put in why it's commented out.